src/Sendy.php

```diff
@@ -17,6 +17,10 @@ class Sendy
      */
     CONST URI_UNSUBSCRIBE = 'unsubscribe';
     /**
+     * Create campaign API URI
+     */
+    CONST URI_DELETE = 'api/subscribers/delete.php';
```

Comment:
Please refactor const name to URI_DELETE_SUBSCRIBER
Const names fits with section in https://sendy.co/api